examples/OpenML_Tutorial.py

```diff
@@ -0,0 +1,307 @@
+"""
+OpenML Introduction
+===================
+
+A tutorial on how to get started on using OpenML.
+"""
+
+# coding: utf-8
+
+# # OpenML in Python 
+# OpenML is an online collaboration platform for machine learning: 
+# 
+# * Find or share interesting, well-documented datasets
+# * Define research / modelling goals (tasks)
+# * Explore large amounts of machine learning algorithms, with APIs in Java, R, Python
+# * Log and share reproducible experiments, models, results 
+# * Works seamlessly with scikit-learn and other libraries
+# * Large scale benchmarking, compare to state of the art
+
+# # Installation
+# 
+# * Up to now: `pip install git+https://github.com/openml/openml-python.git@develop`
+# * In the future: `pip install openml`
+# * Check out the installation guide: [https://openml.github.io/openml-python/stable/#installation](https://openml.github.io/openml-python/stable/#installation)
+
+# ## Authentication
+# 
+# * Create an OpenML account (free) on http://www.openml.org. 
+# * After logging in, open your account page (avatar on the top right)
+# * Open 'Account Settings', then 'API authentication' to find your API key.
+# 
+# There are two ways to authenticate:  
+# 
+# * Create a plain text file `~/.openml/config` with the line 'apikey=MYKEY', replacing MYKEY with your API key.
+# * Run the code below, replacing 'YOURKEY' with your API key.
+
+# In[1]:
```

Comment:
Same here.

Reply:
I thought then when a user would download the notebook from the examples gallery, it would somehow parse the above comments to rebuild the notebook. Just checked and it does not work, so I will remove them.